pom.xml

```diff
@@ -758,6 +758,17 @@
         <artifactId>maven-deploy-plugin</artifactId>
         <version>3.1.0</version>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-gpg-plugin</artifactId>
+        <version>3.0.1</version>
+        <configuration>
+          <gpgArguments>
+            <arg>--pinentry-mode</arg>
```

Comment:
Needed due to ` gpg: signing failed: Inappropriate ioctl for device`: https://github.com/stoyanK7/checkstyle/actions/runs/4204200206/jobs/7294547925#step:6:3125

Answer from: https://stackoverflow.com/a/53992951/9553927